constants.ts

```diff
@@ -0,0 +1,12 @@
+export const SERVER_URL = import.meta.env.VITE_SERVER_URL;
```

Comment:
this constants should be moved to a separate file src/constants/environment.constants.ts
```suggestion
export const API_URL = import.meta.env.VITE_API_URL;
```

Reply:
Fixed